modules/customize-certificates-manually-rotate-service-ca.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-signing-certificates.adoc
+
+[id="manually-rotate-service-ca-{context}"]
+= Manually rotate the service CA certificate
+
+The service CA is valid for one year after {product-title} is 
+installed. If you have a cluster that has been running
+for one year without upgrades, then you will need to manually
+refresh the service CA.
+
+This process is automatically performed during an upgrade,
+and should only be followed for clusters that do not perform
+upgrades.
+
+.Procedure
+
+. View the expiration date of the current service CA certificate by
+using the following command. If this certificate expires
+before a scheduled upgrade, then proceed through the following steps.
+Otherwise, there is no need to perform the rotation at this time.
++
+----
+$ oc get secrets/signing-key -n openshift-service-ca -o template='{{index .data "tls.crt"}}' | base64 -d | openssl x509 -noout -enddate
+----
+
+. Manually rotate the service CA.
++
+----
+$ oc delete secret/signing-key -n openshift-service-ca
+----
+
+. Restart all Pods. This refreshes the created service 
```

Comment:
Might need a little clarification. This reads as if this triggers the actual certificate refresh. It's actually refreshing the pods so that they pick up the newly rotated serving certs.

Reply:
This section has been updated, and now reads as:

> Restart all Pods. This refreshes all Pods so that they receive the newly
> created service certificates created by the recently rotated service CA. 
> This command ensures that all services use the updated certificates.